docs/upgrade/index.md

```diff
@@ -37,12 +37,12 @@ Ensure your operating system is fully up-to-date:
 
 Update the Foreman and Katello release packages:
 
-  * RHEL6 / CentOS 6 **Foreman packages are not available yet**:
+  * RHEL6 / CentOS 6:
 
 ```
 
   # yum update -y http://fedorapeople.org/groups/katello/releases/yum/{{ site.version }}/katello/el6/x86_64/katello-repos-latest.rpm
-  # yum update -y http://yum.theforeman.org/releases/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm
+  # yum update -y http://fedorapeople.org/releases/yum/foreman/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm
```

Comment:
same as above - do  you mean

https://fedorapeople.org/groups/katello/releases/yum/foreman/{{ site.foreman_version }}/el6/x86_64/foreman-release.rpm ?